src/main/java/org/medicmobile/webapp/mobile/Utils.java

```diff
@@ -77,4 +79,15 @@ static void restartApp(Context context) {
 		context.startActivity(intent);
 		Runtime.getRuntime().exit(0);
 	}
+
+	static boolean isConnectionError(WebResourceError error) {
+		switch (error.getErrorCode()) {
+			case ERROR_HOST_LOOKUP:
+			case ERROR_PROXY_AUTHENTICATION:
+			case ERROR_CONNECT:
+			case ERROR_TIMEOUT:
+				return true;
+		}
```

Comment:
I think this is the right collection of errors to catch. There are some others that are maybes for me, but this is fine for now. One concern I have is it'll be hard to check which of these errors a user is getting without checking the log. How do you feel about adding a "more information" button or similar to reveal which error code you got?

Reply:
Added the button in the activity, it shows the error description (though looks more like a code, and the error code number):

![Medic Android - No Internet - More Info](https://user-images.githubusercontent.com/1608415/112204169-212ee500-8bf2-11eb-86cc-c06f6cd79dfa.gif)

We can create a ticket after merge this to make more appealing.

Reply:
Awesome, thanks!